app/build.gradle

```diff
@@ -71,35 +71,36 @@ dependencies {
     kapt libraries.roomProcessor
     implementation libraries.roomRxjava
 
-    implementation libraries.rxjava
-    implementation libraries.rxandroid
+    implementation libraries.rxKotlin
+    implementation libraries.rxAndroid
 
     implementation libraries.moshi
     implementation libraries.okhttp
     implementation libraries.okhttpLogger
 
     implementation libraries.timber
     implementation libraries.threeTenABP
+    implementation libraries.rxBinding
 
     implementation libraries.fresco
     api libraries.frescoOkHttp
     implementation libraries.frescoAnimatedGif
     implementation libraries.frescoWebP
     implementation libraries.frescoAnimatedWebP
 
-    implementation libraries.frescoImageViewer
-
     kapt libraries.kotshiCompiler
     implementation libraries.kotshiApi
 
-    implementation libraries.floatingSearchView
+    implementation libraries.frescoImageViewer
 
     implementation libraries.androidSvg
 
     implementation libraries.aVLoadingIndicatorView
 
     implementation libraries.textDrawable
 
+    implementation libraries.moshiLazyAdapters
```

Comment:
where we are using this?

Reply:
Good catch! This was in use with the SDK (from this PR: https://github.com/RocketChat/Rocket.Chat.Kotlin.SDK/pull/55 - related to FallbackEnum). I declared this dependency here because the app was not able to find it by the SDK. Removing it...